client/reader/following/main.tsx

```diff
@@ -16,9 +18,54 @@ import './style.scss';
 
 function FollowingStream( { ...props } ) {
 	const { currentView } = useFollowingView();
+	const { data: subscriptionsCount, isLoading: isLoadingCount } =
+		SubscriptionManager.useSubscriptionsCountQuery();
+	const { data: siteSubscriptions, isLoading: isLoadingSiteSubscriptions } =
+		SubscriptionManager.useSiteSubscriptionsQuery();
+
+	const isLoading = isLoadingCount || isLoadingSiteSubscriptions;
+
+	const hasNonSelfSubscriptions = useMemo( () => {
+		if ( isLoading ) {
```

Comment:
Why is this check in this function? I think having it within the `hasNonSelfSubscriptions` function muddies the intent of the function, meaning the value really represents `hasNonSelfSubscriptionsAndNotLoadingSubscriptions` 😂.

Can we move the loading check outside of this function? It looks like we're checking `isLoading` again on line 50 so maybe this check is redundant anyway.

Reply:
You're right, it's redundant. I went ahead and removed it. 👍